internal/grpc/server_test.go

```diff
@@ -77,7 +78,7 @@ func TestGRPCStreaming(t *testing.T) {
 		},
 		"StreamEndpoints": func(t *testing.T) {
 			// endpoints will be ignored unless there is a matching service.
-			tr.OnAdd(&v1.Service{
+			et.OnAdd(&v1.Service{
```

Comment:
Will this cause an error in the EndpointsTranslator, given that we are passing a v1.Service?

Reply:
What's more concerning is the test didn't fail ... I'll check it out.